src/test/java/com/google/api/gax/grpc/SettingsTest.java

```diff
@@ -16,8 +16,8 @@
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import org.mockito.Mockito;
+import org.mockito.internal.matchers.apachecommons.ReflectionEquals;
```

Comment:
This import is equivalent in spirit to the shaded import in AutoValue. We should probably explicitly declare a dependency on Apache Commons to pull in the dependency that we want.